README.md

```diff
@@ -1,158 +1,111 @@
 1. [Security Configurations](#security-configurations)
 
-Development: for devs who want to dig into the code, see our hacking [README](hack/README.md#getting-started-for-developers) (WIP)
 
 ## Purpose
 
-This project eases the burden on cluster admins seeking to take advantage of Kubernetes/Openshift orchestration of their virtualized app platforms. For the purposes of running a VM inside a container, the imported file referred to above is a VM image and is considered to be an immutable _golden image_  source for subsequent cloning and instantiation. As a first step in migration to a Kubernetes cluster, virtual machine images must be imported into a location accessible to the kubelet. The Data Importer automates this by copying images from an external http repository and persisting them in in-cluster storage. The components of this process are detailed below.
-
-## Design
-
-The diagram below illustrates the architecture and control flow of this project.
-
-![](doc/diagrams/cdi-controller.png)
-
-### Work Flow
-The agents responsible for each step are identified by corresponding colored shape.
-
-#### Assumptions
-
-- (Optional) A "golden" namespace  which is restricted such that ordinary users cannot create objects within it. This is to prevent a non-privileged user from trigger the import of a potentially large VM image.  In tire kicking setups, "default" is an acceptable namespace.
-
-- (Required) A Kubernetes Storage Class which defines the storage provisioner. The "golden" pvc expects dynamic provisioning to be enabled in the cluster.
-
-#### Event Sequence
-
-1. The admin creates the Controller using a Deployment manifest provided in this repo. The Deployment launches the controller in the "golden" namespace and ensures only one instance of the controller is always running. This controller watches for PVCs containing special annotations which define the source file's endpoint path and secret name (if credentials are needed to access the endpoint).
-
-1. (Optional) If the source repo requires authentication credentials to access the source endpoint, then the admin can create one or more secrets in the "golden" namespace, which contain the credentials in base64 encoding.
-
-1. The admin creates the Golden PVC in the "golden" namespace.  This PVC should either reference a desired Storage Class or fall to the cluster default.  These PVCs, annotated per below, signal the controller to launch the ephemeral importer pod.  
-
-1. When a PVC is created, the dynamic provisioner referenced by the Storage Class will create a Persistent Volume representing the backing storage volume.
+This project is designed with Kubevirt in mind and provides a declarative method of importing VM images into a Kuberenetes cluster. Through this behavior, cluster administrators can build an abstract registry of template images (referred to as "Golden Images" for their role as templates for image clones).
```

Comment:
How about changing _Through this behavior, cluster administrators can build an abstract registry of template images (referred to as "Golden Images" for their role as templates for image clones)._ to _Through this behavior, cluster administrators can build an in-cluster registry of "Golden" images consumable by Kubevirt._?

Not wanting to use **role** in this sentence.